### DISCUSSION TITLE
Template non valido da tutorial

### DISCUSSION THREAD

**Discussion by ymontinaror** [indomus/forum#3080] [HUB: Home Assistant]
Ciao amici ho un problema: seguendo il vostro [tutorial ](https://indomus.it/progetti/arieggiare-o-deumidificare-quando-e-il-momento-calcolare-lumidita-assoluta-con-la-domotica-home-assistant/) noto errori del tipo:

`TemplateError('ValueError: Template error: float got invalid input 'unknown' when rendering template '{% from 'tools.jinja' import umidita_assoluta %} {{ umidita_assoluta('sensor.ble_temperature_a4c1380cc105', 'sensor.ble_humidity_a4c1380cc105') }}' but no default was specified') while processing template 'Template<template=({% from 'tools.jinja' import umidita_assoluta %} {{ umidita_assoluta('sensor.ble_temperature_a4c1380cc105', 'sensor.ble_humidity_a4c1380cc105') }}) renders=4>' for attribute '_attr_native_value' in entity 'sensor.umidita_assoluta_letto'`

non capisco come fare a fixare il problema (mettere un valore di default? ma dove e come si mette?) 
qui il mio codice:

`template:
  sensor:
    - name: "Umidità assoluta letto"
      state: >-
          {% from 'tools.jinja' import umidita_assoluta %}
          {{ umidita_assoluta('sensor.ble_temperature_a4c1380cc105', 'sensor.ble_humidity_a4c1380cc105') }}
      unit_of_measurement: 'g/m³'
      icon: mdi:water-outline
      state_class: measurement`

Potreste aiutarmi per favore? Grazie.

**Comment by Scialla** [comment]
dovresti postare il codice del sensore che hai creato...
aiutati con gli strumenti sviluppatore al tab modelli
il default si aggiunge alla fine della creazione del sensore con `| default(0)` (ad esempio)
